realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -269,6 +270,23 @@ public String toString() {
     }
 
     /**
+     * Check if the RxJava is can be loaded.
+     *
+     * @return true if RxJava dependency exist.
+     */
+    public static synchronized boolean isRxJavaAvailable() {
```

Comment:
`io/realm/RealmConfiguration.java` and `io/realm/rx/RealmObservableFactory.java` are in different package.